### PR TITLE
fix: remove .git after charmil init

### LINKED ISSUE #246: `charmil init` generates broken git repo
## Description

Git repository created by `charmil init` has broken index.

### Steps to reproduce
- `charmil init`
- `git status`

### Expected vs actual behaviour

```
▶ charmil init
GitHub Organization or Username: test
GitHub Repo Name: test
CLI Name: test
Enumerating objects: 161, done.
Counting objects: 100% (161/161), done.
Compressing objects: 100% (97/97), done.
Total 161 (delta 49), reused 141 (delta 34), pack-reused 0
updating starter code with names
Your test CLI has been initialized in this directory.

▶ git status
fatal: unknown index entry format 0x1efa0000
```

### PR DESCRIPTION
**Pull request by ankithans**
Closes #246 

### Description
remove .git folder after charmil init.
tested it.

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change
- [ ] Other (please specify)

### Checklist

- [ ] Documentation added for the feature
- [x] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

File: cli/internal/cmd/init/init.go [hunk 1]
```diff
@@ -139,6 +139,12 @@ func replaceText(templateContext TemplateContext) error {
 		return err
 	}
 
+	// remove .git folder
+	err = os.RemoveAll(path + "/.git")
```
Comment (wtrocki) [inline cli/internal/cmd/init/init.go:143]: this form of concatenation is not preffered due to os. issues. Use path builder instead:

https://pkg.go.dev/path/filepath

**Comment by wtrocki** [comment]
@ankithans Do we have way to do git export (instead of performing git clone)

File: cli/internal/cmd/init/init.go (hunk 1 above)
Comment (ankithans) [reply cli/internal/cmd/init/init.go:143]: sure!! thanks.

**Comment by ankithans** [comment]
I tried to search for it, initially when developing init command. was not able to find anything.
will try to search for it again

**Comment by wtrocki** [comment]
Ok! thank you so much. Let's leave it this way

**Pull request merged**